docs/configuration/recording_rules.md

```diff
@@ -78,6 +78,10 @@ name: <string>
 # How often rules in the group are evaluated.
 [ interval: <duration> | default = global.evaluation_interval ]
 
+# The limit of alerts an alerting rule can produce, or the limit of
```

Comment:
```suggestion
# Limit the number of alerts and series individual rules can produce.
# This is scoped per rule.
```

Reply:
The limit is applied to individual rules, but it's configured for the entire rule group, which makes "This is scoped per rule." confusing.

Reply:
yeah we can omit that sentence. but the first one should make it clear that it is per rule.

Reply:
Although I think it's possible that an alerting rule can produce more series than alerts, and we only limit alerts. Should we also limit series?

Reply:
yes we should limit series